ee/clickhouse/system_status.py

```diff
@@ -80,6 +92,8 @@ def get_clickhouse_slow_log() -> List[Dict]:
             FROM system.query_log
             WHERE query_duration_ms > {SLOW_THRESHOLD_MS}
               AND event_time > %(after)s
+              AND query NOT LIKE 'system.query_log'
```

Comment:
Don't you need a % on this line as well? If I understand correctly, right now it's checking that `query != system.query_log`

Reply:
you're 100% correct

Reply:
Will fix soon - does everything else lgty?

Reply:
Sorry, had a call, looking at the rest below now 😋

Reply:
Yep looks good 👍